.github/workflows/debug-workflow.yml

```
@@ -0,0 +1,36 @@
+
+jobs:
+  debug-info:
+    runs-on: ubuntu-latest
+    if: inputs.debug_enabled || vars.DEBUG_WORKFLOW == 'true'
+    
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Log Branch Info
+        run: |
```

actionlint (1.7.4)

"github.head_ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions for more details

(expression)